webpack.config.js

```diff
@@ -134,26 +132,24 @@ const icoRule = function() {
 };
 
 const output = function() {
-  let output = {
-    publicPath: '/',
-    path: path.resolve(__dirname, './lib'),
-    filename: outputFile,
-    library: libraryName,
+  return {
+    filename: 'europeana-media-player.min.js',
+    library: 'EuropeanaMediaPlayer',
+    libraryExport: 'default',
```

Comment:
This also requires an update in the [README - Embed player](https://github.com/europeana/media-player#embed-player) on how to require the player (leave out the default)